src/app/public/modules/popover/types/popover-trigger.ts

```diff
@@ -1,5 +1,4 @@
 /**
  * Specifies the user action that displays the popover.
- * @deprecated Specifying a trigger type other than 'click' will not be supported in the next major version.
```

Comment:
Why did we just remove the `deprecated` annotation here but leave the property?

Reply:
Same as above.